docs/source/terms_and_model.rst

```diff
@@ -555,13 +564,13 @@ chromosomal features.
   <https://registry.identifiers.org/registry/taxonomy>`__, and the
   CURIE reference MUST be an NCBI taxonomy identifier (e.g., 9606 for
   Homo sapiens).
-* ChromosomeLocation is intended primarily for Humans.  Support for
+* ChromosomeLocation is intended primarily for humans.  Support for
   other species is possible and will be considered based on community
   feedback.
 * `chromosome` is an archetypal chromosome name. Valid values for, and
   the syntactic structure of, `chromosome` depends on the species.
   `chromosome` MUST be an official sequence name from `NCBI Assembly
-  <https://www.ncbi.nlm.nih.gov/assembly>`__.  For Humans, valid
+  <https://www.ncbi.nlm.nih.gov/assembly>`__.  For humans, valid
   chromosome names are 1..22, X, Y (case-sensitive).
```

Comment:
Are we intentionally leaving out MT or M for mitochondrial DNA? Is this because ISCN does not cover Mitochondrial DNA?

Reply:
Yes. Cytogenetics deals only with the genetic content of the cell nucleus, not mitochondria.